File: OCH_Tao/ABC/B/359.js
const input = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
const N = parseInt(input[0], 10);
const A = input[1].split(" ").map(x => parseInt(x, 10));
let cnt = 0;
for (let i = 1; i < N + 1; i++) {
  if (Math.abs(A.indexOf(i) - A.lastIndexOf(i)) === 2) {
    cnt++;
  }
}
console.log(cnt);

File: OCH_Tao/ABC/C/359.py
SX,SY = map(int,input().split())
TX,TY = map(int,input().split())
cnt = 0
cnt+=abs(SY-TY)
if abs(SX-TX)<=cnt:
  print(cnt)
else:
  print(cnt+(abs(SX-TX)-cnt)//2)